garden_ai/notebook_templates/sklearn.ipynb

```diff
@@ -85,13 +128,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {
     "id": "H7em6SwMdvkt"
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "294ca8fae131489498aba71a402e3d0e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "README.md:   0%|          | 0.00/755 [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "my_hugging_face_repo = HFConnector(\"garden-ai/sklearn-iris\")"
+    "my_hugging_face_repo = create_connector(\"garden-ai/sklearn-iris\", \"HF\")\n",
```

Comment:
I think this needs to be `my_hugging_face_repo = create_connector("garden-ai/sklearn-iris", repo_type="HF")`

Reply:
Thanks for catching this! The repo type was positional at first and I forgot to look back at the notebooks after I changed it to a kwarg.